components/04-views/dashboard/default/dashboard.hbs

```diff
@@ -0,0 +1,23 @@
+{{ render '@header' }}
+
+<div class="dashboard container">
+    <div class="grid">
+        <div class="grid__column grid__column--1-4 dashboard__sidebar">
+            {{ render '@dashboard-nav' }}
+
+            {{> '@sidebar-nav-list' wishlist }}
+
+            {{> '@sidebar-nav-list' recentlyOrdered }}
+        </div>
+
+        <div class="grid__column grid__column--3-4 dashboard__content">
+            {{ render '@dashboard-table' }}
+
+            {{> '@dashboard-items' account }}
+
+            {{> '@dashboard-items' addreses }}
+        </div>
+    </div>
+</div>
+
+{{ render '@footer' }}
```

Comment:
To fix:
Sidebar title and main column title should be in the same size and their border bottom should be on the same level on desktop
<img width="1002" alt="screen shot 2017-09-07 at 12 32 08" src="https://user-images.githubusercontent.com/15080214/30159404-bd9d0b2c-93c8-11e7-8eb9-499e565d9204.png">

please make a note (some comments or readme), that class current in sidebar nav is changing by magento

Reply:
changed size to the same + add padding to content to keep it on the same level

Reply:
which class it 'that' ? :)

Reply:
`.cuurent`

Reply:
Comment added